main/depend.common

```diff
@@ -321,6 +321,10 @@ module_bl_mynn_wrapper.o: \
 	module_bl_mynn_common.o 
 
 
+module_bl_gwdo.o: \
+  physics_mmm/bl_gwdo.o
```

Comment:
With depend.common finally having consistent whitespace I think it would be nice to keep the same formatting for new additions